docs/network.md

```diff
@@ -0,0 +1,20 @@
+# 11. Code flow - routing a message
+This is the example of the message that is being sent between nodes (`RawRoutedMessage`) (https://github.com/near/nearcore/blob/fa8749dc60fe0de8e94c3046571731c622326e9f/chain/network-primitives/src/types.rs#L362)
+
+Each of these methods have a `target` - that is either the account_id or peer_id or hash (which seems to be used only for route back...). 
+If target is the account - it will be converted using `routing_table.account_owner` to the peer.
+
+Upon receiving the message, the `PeerManagerActor` will sign it (https://github.com/near/nearcore/blob/master/chain/network/src/peer_manager.rs#L1285)
+And convert into RoutedMessage (which also have things like TTL etc)
+
+Then it will use the `routing_table`, to find the route to the target peer (add `route_back` if needed) and then send the message over the network as `PeerMessage::Routed`.
```

Comment:
Can we add a couple sentences to explain how routing table determines the route to the target peer? If it is already mentioned in other places we can cross link it here.

Reply:
Ok, I added a sentence, and I mentioned that it's in section 8.